### PR TITLE
Misc fixes and improvements

### PR DESCRIPTION
**Pull request by purcell**
A mix of small improvements and fixes. Would close issues #6 and probably #7 too, as well as replacing pull requests #3, #4 and #5.

If you're not maintaining this any more, let me know, and I'll be happy to adopt the package and point the MELPA recipe at my fork instead.

### CODE REVIEW

**Comment by costa** [comment]
@inkel please consider @purcell's proposal, the package has not been touched for 4 (four) years, and it doesn't work with the current Emacs — you've most probably gone to another editor :)

@purcell Thank you for your effort, does your fork currently work with GitHub issues? Do you know by any chance if it is compatible with Spacemacs' github layer? I'm most interested in including the former in the latter.

**Comment by inkel** [comment]
> @inkel please consider @purcell's proposal

Hi! I am considering, I'm just leaving my paternity leave, so catching up with everything.

> you've most probably gone to another editor

That would be treason! :stuck_out_tongue:

@purcell thank you so much for taking an interest in this crazy little idea I once have. Let me review your proposed changes and I'll get back at you. Thanks.

File: github-issues.el [hunk 1]
```diff
@@ -225,8 +225,8 @@
     (message "No current issue selected")))
 
 (defvar github-issues-mode-map
-  (let ((map (make-keymap)))
-    (define-key map "\C-cr" 'github-issues-refresh)
+  (let ((map (make-sparse-keymap)))
+    (define-key map "g" 'github-issues-refresh)
```
Comment (inkel) [inline github-issues.el]: What is the difference between `make-keymap` and `make-sparse-keymap`? `M-x describe-function` wasn't very useful for me.

Comment (purcell) [inline github-issues.el]: `make-keymap` returns a keymap which has empty entries for all keys. `make-sparse-keymap` returns an empty keymap. That's almost always what you want.

Comment (inkel) [inline github-issues.el]: Thanks for the explanation.

**Comment by purcell** [comment]
Yay, thanks!

**Comment by Silex** [comment]
@purcell, @inkel: I think my PR contained something somewhat important that was not included in purcell's PR: https://github.com/Silex/github-issues.el/commit/5491a5462ff35493f0fb8dcb92dd131cf50c9d67.diff

That is, sanitize the `replace-string` that uses a literal newline instead of an escaped one.

@inkel: also, please consider searching for a new maintainer if you're not interested in maintening anymore. Maybe that @emacsmirror could work.

**Comment by purcell** [comment]
> That is, sanitize the replace-string that uses a literal newline instead of an escaped one.

Yes, good call.

> @inkel: also, please consider searching for a new maintainer if you're not interested in maintening anymore. Maybe that @emacsmirror could work.

One good way to do this, IMO, is to simply add contributors like @Silex or myself as collaborators on the github project so they can occasionally help out. This works well for lots of projects.

**Comment by Silex** [comment]
> One good way to do this, IMO, is to simply add contributors like @Silex or myself as collaborators on the github project so they can occasionally help out. This works well for lots of projects.

Yeah, good idea :+1:

**Comment by inkel** [comment]
@Silex @purcell great suggestion, I'll add you both as maintainers. Thanks!

**Comment by Silex** [comment]
:+1:

**Comment by purcell** [comment]
Yay, thanks!

**Pull request merged**